### DISCUSSION TITLE
Erro 404 no Upload de arquivos

### DISCUSSION THREAD

**Discussion by jordanrafaell** [community/community#68855] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Sempre que subo um arquivo para o github, o site não vai por completo! um dos aquivos não carregou e aparece o erro 404 quando vejo no NetWork! alguem sabe como resolver? ja fiz de tudo e só resolve uma vez ou outra.

**Comment by OshandaB** [comment]
Response #1

**Comment by seyLu** [comment]
#### Hi @jordanrafaell,

Assets are loaded just fine.
![image](https://github.com/community/community/assets/98249191/896acbbc-bf7c-49f4-9b3d-36ad08629b72)

Change links from absolute to relative links: 
![image](https://github.com/community/community/assets/98249191/1692b3ba-e118-4705-9c43-271615a438b2)

```diff
- <link rel="stylesheet" href="/assets/styles/main.css">
+ <link rel="stylesheet" href="assets/styles/main.css">
```
Do that for every link.

Also add a base href pointing to the base repo route:
```diff
<head>
  ...
+ <base href="/Pokemon.github.io/">
 </head>
```
